shared/fs/common/hooks.tsx

```diff
@@ -42,7 +42,9 @@ const useFsNonPathSubscriptionEffect = (topic: RPCTypes.SubscriptionTopic) => {
   React.useEffect(() => {
     const subscriptionID = Constants.makeUUID()
     dispatch(FsGen.createSubscribeNonPath({subscriptionID, topic}))
-    return () => dispatch(FsGen.createUnsubscribe({subscriptionID}))
```

Comment:
newer lint wants us to return void and not what dispatch returns, which is what we want anyways